Cql/Cql.Runtime/ValueSets/CqlValueSetFacade.cs

```diff
@@ -46,39 +45,14 @@ public CqlValueSetFacade(string id, IValueSetDictionary valueSets)
         private readonly string _canonical;
         private readonly IValueSetDictionary _valuesets;
 
-        /// <summary>
-        /// Returns <see langword="true"/> if <paramref name="code"/> is in this value set.
-        /// </summary>
-        /// <param name="code">The code to check.</param>
-        /// <returns><see langword="true"/> if <paramref name="code"/> is in this value set.</returns>
-        public bool? IsCodeInValueSet(CqlCode? code)
-        {
-            if (code == null || code.code == null)
-                return null;
-            if (code.system != null)
-            {
-                var @in = _valuesets.IsCodeInValueSet(_canonical, code.code, code.system);
-                return @in;
-            }
-            else
-            {
-                var @in = _valuesets.IsCodeInValueSet(_canonical, code.code);
-                return @in;
-            }
-        }
+        /// <inheritdoc/>
+        public bool IsCodeInValueSet(CqlCode code) => _valuesets.IsCodeInValueSet(_canonical, code);
 
-        /// <summary>
-        /// Returns <see langword="true"/> if <paramref name="code"/> is in this value set.
-        /// </summary>
-        /// <param name="code">The code to check.</param>
-        /// <param name="system">The code system to check.</param>
-        /// <returns><see langword="true"/> if the code is in this value set.</returns>
-        public bool? IsCodeInValueSet(string? code, string? system)
-        {
-            if (code == null || system == null)
-                return null;
-            else return _valuesets.IsCodeInValueSet(_canonical, code, system);
-        }
+        /// <inheritdoc/>
+        public bool IsCodeInValueSet(string code, string? system) => _valuesets.IsCodeInValueSet(_canonical, code, system);
+
+        /// <inheritdoc/>
+        public bool IsCodeInValueSet(string code) => _valuesets.IsCodeInValueSet(_canonical, code);
```

Comment:
There was divergent behaviour for how to handle a system being null. In some implementations this meant: find a code with no system, in other implementations of this interface (and IValueSetDictionary) it meant: find any code that matches. I have split these up into two separate methods.